src/modules/peek/Peek.UI/PeekXAML/MainWindow.xaml.cs

```diff
@@ -72,12 +74,17 @@ private void PeekWindow_Activated(object sender, Microsoft.UI.Xaml.WindowActivat
         {
             if (args.WindowActivationState == Microsoft.UI.Xaml.WindowActivationState.Deactivated)
             {
+                // TitleBarControl.FileCountAndNameContainer.AppTitle_FileName.Foreground = (SolidColorBrush)App.Current.Resources["WindowCaptionForeground"];
```

Comment:
Easier to create a `Deactivated` visual state in `TitleBar`. You can then set the state here if needed, and all the visual changes would live there. In this way you don't need to set brushes in code-behind (you'd need to update those again whenever the theme changes - there is no "ThemeResource" equivalent in C#, only in XAML)

Reply:
- don't think this class has access to window properties
- would like to follow MS examples

Reply:
Yeah, see my earlier comment on maybe waiting until the Titlebar control is out and let this handle it.

The samples are samples.. not always 100% correct :). Setting ThemeResources in C# is something you'd want to avoid.

Reply:
I know there are different ways to achieve the same result. But please, I'm asking you, don't make it more difficult than it already is and please let me follow a MS example?

P.S. in case it wasn't clear: MainWindow and TitleBar are two different scopes/classes that do not have acces to each other.

Reply:
> Easier to create a `Deactivated` visual state

You mean like in WinUI Gallery? That doesn't even work!

https://github.com/microsoft/WinUI-Gallery/blob/1ffc48c43b9b235c299a95d01d7d2458ee713daa/WinUIGallery/Navigation/NavigationRootPage.xaml#L43-L50

Reply:
See the Titlebar control in Labs as a reference, not WinUI Gallery.

My point is setting the brush in code-behind will not make the brush theme aware. That means if the theme changes (or in high contrast) the incorrect color will be shown - resulting in a11y issues and flags, things that need additional fixing.

Again, maybe just wait for the TitleBar control to be available and not worry about this at all?

Reply:
I've used this in a few other windows lately and it works fine! Check Hosts file or RegPreview. (Well, it only needs _one_ active/deactive switch, but that's pretty acceptable to me.)

> Again, maybe just wait for the TitleBar control to be available and not worry about this at all?

How soon again? That would be a good pr for you, adressing multiple windows at once.

Reply:
Maybe I wasn't clear - it's no adittional work and makes things easier for us going forward. 

Yes it works, but when changing theme/mode it would not update to the correct color in the new mode. That's because the brush is set in code and would ideally need to be updated on a theme changing event. When defined in XAML as a `ThemeResource`, this would be handled automatically for us! (And we could very easily disable all interactive controls as well, which is recommended in a deactivated state).

```xml
<VisualStateGroup x:Name="WindowActivationStates">
      <VisualState x:Name="Activated" />
      <VisualState x:Name="Deactivated">
            <VisualState.Setters>
                  <Setter Target="AppTitle_FileName.Foreground" Value="{ThemeResource TextFillColorDisabledBrush}" /> 
                  <!-- Could disable the pin and open with buttons here as well? -->
            </VisualState.Setters>
      </VisualState>
</VisualStateGroup>
```

You can then call the state in the activated/deactivated event like this:

```csharp
VisualStateManager.GoToState(this, "Deactivated", true); // This is the name of the `VisualState`
```

Reply:
Thanks for the suggestion. I'm convinced it already works. I really want to focus on getting "the merge" to work. Please.